app/main.py

```diff
@@ -1,8 +1,21 @@
 class Person:
-    # write your code here
-    pass
+    people = {}
```

Comment:
The `people` dictionary is defined as a class variable, but it is being accessed as an instance variable in the `__init__` method. You should use `Person.people` to reference the class variable.